panel/pane/deckgl.py

```diff
@@ -24,6 +26,7 @@
 if TYPE_CHECKING:
     from bokeh.document import Document
     from bokeh.model import Model
+    from bokeh.models import DataDict
```

Comment:
@mattpap, at the top of your head do you know if this has been available since Bokeh 3.5?

Reply:
This predates bokeh 3.0.